loki.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+die() {
+  local msg=$1
+  local code=${2-1} # default exit status 1
+  msg "$msg"
+  exit "$code"
+}
+
+setup_colors() {
+  if [ -t 2 ] && [ "${TERM-}" != "dumb" ]; then
+    NOFORMAT='\033[0m' RED='\033[0;31m' GREEN='\033[0;32m' ORANGE='\033[0;33m' BLUE='\033[0;34m' PURPLE='\033[0;35m' CYAN='\033[0;36m' YELLOW='\033[1;33m'
+  else
+    NOFORMAT='' RED='' GREEN='' ORANGE='' BLUE='' PURPLE='' CYAN='' YELLOW=''
+  fi
+}
+setup_colors
+
+msg() {
+  echo >&2 -e "${1-}"
+}
+
+run_storybook() {
+  args=("$@")
+
+  if [ ${#args[@]} -eq 0 ]; then
+    args=''
+  else
+    args=":$@"
+  fi
+
+  yarn storybook$args --no-open > /dev/null 2>&1 &
+  SB_PID=$!
+  msg "${NOFORMAT}--- Starting ${CYAN}yarn storybook${args}${NOFORMAT} with process id ${GREEN}${SB_PID}${NOFORMAT}"
+}
+
+run_storybook "$@"
+
+# wait for storybook to be ready
+attempts=0
+until $(curl --output /dev/null --silent --head --fail http://localhost:6006); do
+  if [ $attempts -eq 10 ]; then
+    kill $SB_PID
+    die "[x] something went wrong starting storybook .."
+  fi
+  attempts=$(($attempts+1))
+  sleep 2
+done
+
+# storybook is up, start the appropriate loki process
+msg "[${GREEN}✓${NOFORMAT}] Storybook is up!"
+msg "${NOFORMAT}--- Starting ${CYAN}yarn loki:update${args}${NOFORMAT}"
+
+if yarn loki:update$args ; then
```

Comment:
What if we pass extra args through to loki like `test` and `update`? So you could use it for both loki commands like `./loki.sh test` or `./loki.sh update` or `./loki.sh update --healthcare`? I guess there's a variety of ways it could be structured, but we have two use cases we can cover over three brands ({test, update} for {core, healthcare, medicare})

Reply:
Actually, there's also a an `approve` command, which prunes old images. So this command could cover {test, update, approve} for {core, healthcare, medicare}.

Reply:
Nice, thanks for the feedback! Yes this sounds like a good set of expansions.. easy enough to add flags, could probably even run multiple sequentially that would be nice.. I'll dig into it :construction\_worker:

Reply:
Made these updates, removed color (made code messy, not needed), run it by itself for usage.. updated test in pr desc